### ISSUE TITLE
Does include really accept variables?

### ISSUE THREAD

**Issue by derekm** [tj/ejs#218]
Docs for this project kinda suck. Docs for `include` seem to accidentally indicate that `include` accepts variables, but I can't get it to work.

See the misleading docs here: https://github.com/tj/ejs#includes

Ignore the explanation and focus on the code. The code tricks you into thinking "user" is a variable in the `include` line. Is it? I think it isn't. Why is the code example so misleading, arbitrary and confusing-in-intention?

That said: Variables in `include` would be really nice.

**Comment by mde** [comment]
This project is no longer maintained -- the repo is only here for historical reasons. The current version of EJS is here: https://github.com/mde/ejs / http://ejs.co/

**Comment by derekm** [comment]
Ha! Thanks! I realized that after commenting!

Docs still suck in the new project, but the includes explanation is more complete and indicates that "user" is meant to be a template variable instead of a path variable.

FYI, using the new project codebase, I was able to push function definitions into the view variables and use them in EJS templates to resolve dynamic paths and pass those paths to include.

PS: Thanks for your overall commitment to open source!
